app/views/kaminari/_prev_page.html.erb

```diff
@@ -1,9 +1,9 @@
 <span class="prev">
   <% unless current_page.first? %>
```

Comment:
Just checking – is it intentional to generate the empty `span class="prev"` when it's the first page?